jenkins/spark-premerge-build-ut.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
can we just put the content into spark-premerge-build.sh and use the flag/ENV to control if run?  so people do not need to find 2 places to make changes

Reply:
here just try to keep it simple. So people can easily run IT or UT separately. I can't think of any case to make change 2 places at the same time.

BTW, `. jenkins/version-def.sh` would be removed as it's not required by UT

Reply:
I don't think people would like to use this script to run test locally, it was for pre-merge only

Reply:
OK, then I would suggest just move to Jenkinsfile as it's quite simple. And it would be easier to run different version UT in parallel if necessary in the future